sierra/src/main/java/module-info.java

```diff
@@ -0,0 +1,6 @@
+module sierra {
```

Comment:
Based on this SO article, it seems that the module name should be `org.httprpc.sierra`:

https://stackoverflow.com/questions/43192741/how-should-i-name-my-java-9-module